nextflow.config

```diff
@@ -51,26 +51,29 @@ params {
-    cf_contamination_adjustment   = false // by default we are not using this in Control-FREEC
-    cf_mincov                     = 0     // ControlFreec default values
-    cf_minqual                    = 0     // ControlFreec default values
-    cf_window                     = null  // by default we are not using this in Control-FREEC
-    cnvkit_reference              = null  // by default the reference is build from the fasta file
-    concatenate_vcfs              = false // by default we don't concatenate the germline-vcf-files
-    ignore_soft_clipped_bases     = false // no --dont-use-soft-clipped-bases for GATK Mutect2
-    wes                           = false // Set to true, if data is exome/targeted sequencing data. Used to use correct models in various variant callers
-    joint_germline                = false // g.vcf & joint germline calling are not run by default if HaplotypeCaller is selected
-    joint_mutect2                = false // if true, enables patient-wise multi-sample somatic variant calling
-    sentieon_haplotyper_emit_mode = "variant" // default value for Sentieon haplotyper
+    ascat_ploidy                      = null  // default value for ASCAT
+    ascat_min_base_qual               = 20    // default value for ASCAT
+    ascat_min_counts                  = 10    // default value for ASCAT
+    ascat_min_map_qual                = 35    // default value for ASCAT
+    ascat_purity                      = null  // default value for ASCAT
+    cf_ploidy                         = "2"   // default value for Control-FREEC
+    cf_coeff                          = 0.05  // default value for Control-FREEC
+    cf_contamination                  = 0     // default value for Control-FREEC
+    cf_contamination_adjustment       = false // by default we are not using this in Control-FREEC
+    cf_mincov                         = 0     // ControlFreec default values
+    cf_minqual                        = 0     // ControlFreec default values
+    cf_window                         = null  // by default we are not using this in Control-FREEC
+    cnvkit_reference                  = null  // by default the reference is build from the fasta file
+    concatenate_vcfs                  = false // by default we don't concatenate the germline-vcf-files
+    ignore_soft_clipped_bases         = false // no --dont-use-soft-clipped-bases for GATK Mutect2
+    joint_germline                    = false // g.vcf & joint germline calling are not run by default if HaplotypeCaller is selected
+    joint_mutect2                     = false // if true, enables patient-wise multi-sample somatic variant calling
+    only_paired_variant_calling       = false // if true, skips germline variant calling for normal-paired sample
+    sentieon_dnascope_emit_mode       = "variant" // default value for Sentieon dnascope
+    sentieon_dnascope_model           = "https://s3.amazonaws.com/sentieon-release/other/SentieonDNAscopeModel1.1.model"
```

Comment:
was this the s3 thing you guys mentioned or is this something new?

Reply:
Yeah, I believe Maxime is working on getting that file into igenomes.